doc/nrf/getting_started/modifying.rst

```diff
@@ -269,7 +269,9 @@ See :ref:`zephyr:setting_configuration_values` in the Zephyr documentation for i
 
    .. group-tab:: Command line
 
-      |west_build_option|
+      If you work on the command line, pass the additional options to the ``west build`` command.
+      The options must be added after a ``--`` at the end of the command.
+      See :ref:`zephyr:west-building-cmake-config` for more information.
```

Comment:
@nordicjm , @mbolivar-nordic , can you confirm this change makes sense?
I found it weird to link to "One-time CMake arguments" under a section called "Permanent changes", so I changed this link to point to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/develop/west/build-flash-debug.html#permanent-cmake-arguments

Reply:
Deferring to Marti, I do not use west.

Reply:
Do not the Permanent Changes refer to Kconfig and not CMake?